packages/Popover/src/components/Trigger/Trigger.js

```diff
@@ -49,9 +50,12 @@ function Trigger(props) {
   const { children, a11yText, ...moreProps } = props;
 
   if (typeof children === "function") {
+    console.log("isoPEN", isOpen);
     return (
       <React.Fragment>
-        {React.cloneElement(children(handleTriggerEvent, { "aria-describedby": content.ariaId }), { ...moreProps })}
+        {React.cloneElement(children(handleTriggerEvent, { "aria-describedby": content.ariaId }, isOpen), {
```

Comment:
Added it as 3rd param to avoid breaking current usage like this:
```
        <Popover.Trigger>
          {(handler, attributes) => (
            <Icon
              {...attributes}
              onMouseOver={handler}
              onMouseOut={handler}
              onFocus={handler}
              onBlur={handler}
              tabIndex={0}
              role="img"
              aria-label="info"
            />
          )}
        </Popover.Trigger>
```